### PR TITLE
[MetaSchedule] Add the missing `HasWorkload` interface to the Database

### PR DESCRIPTION
**Pull request by junrushao1994**
This PR add a previously missing interface `HasWorkload` to the database interface.

CC: @yzhliu @Laurawly @FrozenGene

### CODE REVIEW

File: tests/python/unittest/test_meta_schedule_task_scheduler.py [hunk 1]
```diff
@@ -140,6 +140,9 @@ def __init__(self):
         self.records = []
         self.workload_reg = []
 
+    def has_workload(self, mod: IRModule) -> bool:
+        return False
```
Comment (FrozenGene) [inline tests/python/unittest/test_meta_schedule_task_scheduler.py:144]: Should we query `workload_reg` here instead of `return False` simply?

Comment (junrushao1994) [reply tests/python/unittest/test_meta_schedule_task_scheduler.py:144]: Aha this is a dummy database for mock testing

**Review by FrozenGene — APPROVED** [review]

**Comment by FrozenGene** [comment]
Merged now. @junrushao1994

**Comment by junrushao1994** [comment]
Thank you so much for your time!! @FrozenGene

**Pull request merged**